instrumentation/src/androidTest/java/com/bumptech/glide/NonBitmapDrawableResourcesTest.java

```diff
@@ -17,10 +17,10 @@
 import android.graphics.drawable.Drawable;
 import android.net.Uri;
 import androidx.test.core.app.ApplicationProvider;
-import androidx.test.ext.junit.runners.AndroidJUnit4;
 import com.bumptech.glide.load.resource.bitmap.RoundedCorners;
 import com.bumptech.glide.test.ResourceIds;
 import com.bumptech.glide.testutil.TearDownGlide;
```

Comment:
**Code Structure Issue**: The import 'com.google.common.collect.ImmutableList' is added, which introduces a new dependency. Ensure this dependency is necessary and properly documented. <br> **Fix**: If 'ImmutableList' is essential for the test, keep the import and document its usage. Otherwise, consider using native Java or Android SDK alternatives. <br> **Code Suggestion**: 
 ```
 -import com.google.common.collect.ImmutableList;
 +// Consider using native Java or Android SDK alternatives
 ```